diff --git a/node_modules/pkg-fetch/lib-es5/index.js b/node_modules/pkg-fetch/lib-es5/index.js
index 4b1339e..08d09ee 100644
--- a/node_modules/pkg-fetch/lib-es5/index.js
+++ b/node_modules/pkg-fetch/lib-es5/index.js
@@ -79,7 +79,7 @@ function download(_a, local) {
         return __generator(this, function (_c) {
             switch (_c.label) {
                 case 0:
-                    url = "https://github.com/vercel/pkg-fetch/releases/download/" + tag + "/" + name;
+                    url = "https://hub.gitmirror.com/https://github.com/vercel/pkg-fetch/releases/download/" + tag + "/" + name;
                     _c.label = 1;
                 case 1:
                     _c.trys.push([1, 4, , 5]);
