diff --git a/node_modules/tkserver/index.js b/node_modules/tkserver/index.js
index d1fb7c9..a0972a2 100644
--- a/node_modules/tkserver/index.js
+++ b/node_modules/tkserver/index.js
@@ -210,7 +210,7 @@ function anonymousSignIn (request) {
 
 async function connectToDatabase () {
   if (db) return db
-  const dataDir = path.resolve(process.cwd(), process.env.TWIKOO_DATA || './data')
+  const dataDir = path.resolve(process.pkg ? path.join(process.execPath, '..') : process.cwd(), process.env.TWIKOO_DATA || './data')
   if (!fs.existsSync(dataDir)) {
     fs.mkdirSync(dataDir)
   }
